operations/resourcefeatures.md

```diff
@@ -0,0 +1,193 @@
+    "Client": "Sample Client 1.0.0",
+    "EnterpriseIds": 
+    [
+        "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "4d0201db-36f5-428b-8d11-4f0a65e960cc"
+    ],
+    "ResourceFeatureIds": [
+        "a693dd8c-21fe-4dae-b450-ea3bd9ab3bb0"
+    ],
+    "ResourceFeatureAssignmentIds": [
+        "ca2b5bf9-24f5-4faa-95ef-b65d38598b08"
+    ],
+    "UpdatedUtc": {
+        "StartUtc": "2023-11-01T00:00:00Z",
+        "EndUtc": "2023-11-30T00:00:00Z"
+    },
+    "ActivityStates": [ "Active" ],
+    "Limitation":{
+        "Count": 10
+    }   
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns the resource feature assignments for all enterprises within scope of the Access Token. |
+| `ResourceFeatureIds` | array of string | required, max 1000 items | Unique identifiers of [Resource feature](#resource-feature) to which the resource feature assignments belong. |
+| `ResourceFeatureAssignmentsIds` | array of string | optional, max 1000 items | Unique identifiers of [Resource feature assignment](resourcecategories.md#resource-category-assignment). |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the resource feature assignment were updated. |
+| `ActivityStates` | array of string [Activity state](_objects.md#activity-state) | optional | Whether to return only active, only deleted or both records. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of resource feature assignments returned. |
+
+### Response
+
+```javascript
+{
+    "ResourceFeatureAssignments": [
+        {
+            "Id": "ca2b5bf9-24f5-4faa-95ef-b65d38598b08",
+            "IsActive": true,
+            "ResourceId": "18019693-c66f-4be8-a893-c3d89fd291cc",
+            "FeatureId": "a693dd8c-21fe-4dae-b450-ea3bd9ab3bb0",
+            "CreatedUtc": "2023-11-03T00:00:00Z",
+            "UpdatedUtc": "2023-11-21T00:00:00Z"
+        }
+    ],
+    "Cursor": "3d5201ad-4a7b-4cbe-9c22-03dd79c28443"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ResourceFeatureAssignments` | array of [Resource feature assignment](#resource-feature-assignment) | optional | Resource feature assignments. |
+| `Cursor` | string | required | Unique identifier of the last and hence oldest resource features returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older resource feature. |
```

Comment:
```suggestion
| `Cursor` | string | required | Unique identifier of the last and hence oldest resource feature assignments returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older resource feature assignments. |
```